.github/workflows/main.yml

```diff
@@ -269,6 +269,49 @@ jobs:
       env:
         RUST_BACKTRACE: 1
 
+  # Perform all tests (debug mode) for `wasmtime` with the experimental x64
+  # backend. This runs on the nightly channel of Rust (because of issues with
+  # unifying Cargo features on stable) on Ubuntu.
+  test_x64:
+    name: Test x64
```

Comment:
Could this include something about "new backend" in case anyone gets confused that we're only testing x64 in one place?